shotover-proxy/src/transforms/redis/sink_cluster.rs

```diff
@@ -97,6 +83,36 @@ pub struct RedisSinkCluster {
 }
 
 impl RedisSinkCluster {
+    pub fn new(
+        first_contact_points: Vec<String>,
+        connection_count: usize,
+        tls: Option<TlsConfig>,
+        chain_name: String,
+    ) -> Result<Self> {
+        register_counter!("redis_cluster_failed_request", Unit::Count, "chain" => chain_name);
```

Comment:
Just checking that we want two ways of passing the chain name into this transform? (Task local variable used to get the name for incrementing this counter.) Was there a blocker using the existing method? Or do we want to get rid of the task local variable in the future?